analysis_options.yaml

```diff
@@ -24,7 +24,6 @@
 
 analyzer:
   language:
-    enableStrictCallChecks: true
```

Comment:
?

Reply:
This option is no longer supported, so it has to be removed from the yaml file.